src/olympia/constants/base.py

```diff
@@ -78,10 +78,12 @@
 }
 
 UPLOAD_SOURCE_DEVHUB = 1
-UPLOAD_SOURCE_API = 2
+UPLOAD_SOURCE_SIGNING_API = 2
+UPLOAD_SOURCE_ADDON_API = 3
 UPLOAD_SOURCE_CHOICES = (
     (UPLOAD_SOURCE_DEVHUB, _('Developer Hub')),
-    (UPLOAD_SOURCE_API, _('API')),
+    (UPLOAD_SOURCE_SIGNING_API, _('Signing API')),
+    (UPLOAD_SOURCE_ADDON_API, _('Addon API')),
```

Comment:
What's the difference between the two?

Reply:
`UPLOAD_SOURCE_SIGNING_API` is for uploads that came from the existing signing API, `UPLOAD_SOURCE_ADDON_API` is for uploads that will come from the new add-on submission api